Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -3359,7 +3359,7 @@
   - type: HTN
     rootTask:
       task: RuminantCompound
-      
```

Comment:
untouch

Reply:
yeah, there's nothing I can really do about it